pkgs/development/ocaml-modules/raven/default.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDunePackage,
+  raven-fehu,
+  raven-hugin,
+  raven-kaun,
+  raven-nx,
+  raven-nx-datasets,
+  raven-rune,
+  raven-saga,
+  raven-sowilo,
+  raven-talon,
+}:
+
+buildDunePackage (finalAttrs: {
+  pname = "raven";
+  version = "1.0.0_alpha2";
+
+  src = fetchFromGitHub {
+    owner = "raven-ml";
+    repo = "raven";
+    tag = finalAttrs.version;
+    hash = "sha256-8OIlmJ6iaeNj5DnKSvFEBljyMgDj0dh8DCq4esk68Bc=";
+  };
+
+  # remove docs to prevent pulling unnecessary dependencies during check phase
+  postUnpack = ''
+    rm -r $sourceRoot/{book,www}
+  '';
+
+  propagatedBuildInputs = [
```

Comment:
This *feels* like we have circular dependencies since all the `raven-*` packages are using `src`, `version` and `postUnpack` from this package.

I'm not sure if this is a OK pattern, I'll defer to @vbgl